### ISSUE TITLE
add node quality to Registration Entry protobuf

### ISSUE THREAD

**Issue by blaggacao** [spiffe/spire#1867]
##### TLDR: `parent_id: spiffe://trustdomain/spire/server` -> then it's a node

Shall this be documented?

---

https://github.com/spiffe/spire/blob/06343d63af8bd4b314442337f30467ee38b72f1a/proto/spire/common/common.proto#L58-L85

- does not include the `node` attribute
- json is unmarshalled onto this protobuf struct
- hence, **_there is no directly supported way to decoratively describe node identities_**

As a workaround:
- I need to parse the json file
- detect missing `parent_id` on entries
- infer &mdash; somewhat brittle &mdash; that a missing `parent_id` is no mistake, but a node entry
- filter them out
- subtract them from the other entry set
- launch a command for the remaining normal entries
- launch a command with `-node` flag for the extracted node entries.

----

https://github.com/spiffe/spire/blob/06343d63af8bd4b314442337f30467ee38b72f1a/cmd/spire-server/cli/entry/create.go#L171-L179

Looks like this is just a very simple helper for setting `parent_id` to `[...]/spire/server`
